### DISCUSSION TITLE
版本发布时间问题

### DISCUSSION THREAD

**Discussion by Chiloven945** [Meloong-Git/PCL#4022] [提问] [answered]
感觉每次mc新版本发布时间都很奇怪，按理来说应该是使用EST，但是似乎使用的却是UTC+0……

**Comment by WForst-Breeze** [comment, marked as answer]
Mojang 发布新版本确实用的是 UTC，PCL 会将其转换为北京时间。可能还是为了国际化标准化一点？毕竟是叫 “国际标准时间”（
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/f83051ed-61b4-4577-a300-2e16969d50be)
